app/src/main/java/com/bashkevich/androidfundamentals/moviesdetails/view/FragmentMoviesDetails.kt

```diff
@@ -54,27 +54,31 @@ class FragmentMoviesDetails : Fragment() {
 
         setUpActorsRecyclerView()
 
-        movieId?.let { moviesDetailsViewModel.getMovieFromList(it) }
+        movieId?.let { moviesDetailsViewModel.loadMovie(it) }
 
         moviesDetailsViewModel.movieLiveData.observe(this.viewLifecycleOwner) { selectedMovie ->
             selectedMovie?.let { movie ->
-                Picasso.get().load(Uri.parse(movie.backdrop)).into(backdropView)
+
+                backdropView.load("${RetrofitModule.IMAGES_BASE_URL}${movie.backdrop}") {
```

Comment:
Better to have some function with meaningful name. Or on the step on conversion already have correct url calculation